ccc-js/src/main/javascript/package-res/ccc/core/cartesian/axis/cart-axis.js

```diff
@@ -114,6 +117,10 @@ def('pvc.visual.CartesianAxis', pvc_Axis.extend({
             }
             return ratio;
         },
+
+        _getTickAlignment: function() {
+          return this.ticks && this.scale.type === 'timeSeries' ? this.ticks[0] : this._state.tickAlignment;
```

Comment:
`!== 'discrete'` is best no? Numeric ticks can also benefit from this no?
Also, if no ticks or correct scale type, why not simply return `null` or `undefined`?

Reply:
👍